ghcide/src/Development/IDE/GHC/Compat/Core.hs

```diff
@@ -491,7 +491,11 @@ import           GHC.Unit.Module              hiding (ModLocation (..), UnitId,
                                                toUnitId)
 import qualified GHC.Unit.Module              as Module
 import           GHC.Unit.State               (ModuleOrigin (..))
-import           GHC.Utils.Error              (Severity (..), emptyMessages)
+import           GHC.Utils.Error              (Severity (..), emptyMessages
+#if MIN_VERSION_ghc(9,3,0)
```

Comment:
This does not follow the note `Guidelines For Using CPP In GHCIDE Import Statements`

Reply:
fixed.